### PR TITLE
Migrate frontend files

### PR DESCRIPTION
**Pull request by darwintree**
Migrate frontend code to this repo

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-fans/tokenbatchsender/1)

### CODE REVIEW

File: frontend/src/task.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+0x1e0cc11e4dc7208e74e20ce3060fdffc88680514, 1300
```
[the remark this answers is not in the document]
Comment (darwintree) [reply frontend/src/task.csv]: task.csv is moved into an example folder for development. Related csv format is illustrated in README.md

File: frontend/src/contracts/contracts-config.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/*
+  json为编译合约后生成的，包括bytecode与abi
+  还需要手动指定合约的地址
+*/
+
+// 测试环境可以将引入路径设置为编译文件生成的绝对路径
+import {default as GLD} from "./GLDToken.json";
```
[the remark this answers is not in the document]
Comment (darwintree) [reply frontend/src/contracts/contracts-config.js:7]: This problem will be fixed in a later edition where the config mechanism will be changed.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,94 @@
...
+The `.json` files in `./build/contracts` can be deployed to conflux network (either mainnet or testnet) directly. `DMDToken.json` and `GLDToken.json` are ERC777 sample contracts. `TransferToken.json` takes an address array `trusted_contracts` as the constructor parameter from where the token batch sending request will be accepted.
+
+## Dapp
+
+### Manully change Dapp config file
+
+Suppose you have get the deployed address in [Deploy contracts](#deploy-contracts)
+
+``` javascript
+//frontend/src/contracts-config.js
```
Comment (wangdayong228) [inline README.md:53]: It's not necessary to introduce detail how to use for developers. And the supported token list should set in a config file

File: README.md [hunk 4]
```diff
@@ -1,2 +1,94 @@
...
+
+// import ERC777 Token .json files
+import {default as GLD} from "../../../build/contracts/GLDToken.json";
+import {default as DMD} from "../../../build/contracts/DMDToken.json";
+
```
Comment (wangdayong228) [inline README.md:58]: Only import one erc777.json is enough, create all erc777 tokens use the same abi

Comment (darwintree) [reply README.md]: Resolved in latest version

File: frontend/src/contracts/contracts-config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (darwintree) [reply frontend/src/contracts/contracts-config.js]: Solved in latest version

**Review by wangdayong228 — APPROVED** [review]
good job

**Pull request merged**